webui_runner/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:bionic
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+	apt-utils build-essential gpg-agent \
+	curl ca-certificates wget software-properties-common \
+    psmisc lsof git cmake \
+    npm nano
+
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|apt-key add -
+
+RUN add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main"
+
+RUN apt-get update \ 
+	&& apt-get install -y --no-install-recommends \ 
+	clang-8 lldb-8 lld-8
+
+RUN curl -sL https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-6/libclang_rt.builtins-wasm32-wasi-6.0.tar.gz  | tar x -zf - -C /usr/lib/llvm-8/lib/clang/8.0.1
+
+RUN ln -s /usr/bin/wasm-ld-8 /usr/bin/wasm-ld
+
+ENV LD_LIBRARY_PATH=/usr/local/lib
+
+RUN curl -sS -L -O https://static.rust-lang.org/dist/rust-1.37.0-x86_64-unknown-linux-gnu.tar.gz \
+	&& tar xzf rust-1.37.0-x86_64-unknown-linux-gnu.tar.gz \
+	&& cd rust-1.37.0-x86_64-unknown-linux-gnu \
+	&& ./install.sh \
+	&& cd .. \
+	&& rm -rf rust-1.37.0-x86_64-unknown-linux-gnu rust-1.37.0-x86_64-unknown-linux-gnu.tar.gz
```

Comment:
A thought: we could use an `ARG` variable with a default to factor out the `1.37.0` to a single location and make it easier to update later.